### ISSUE TITLE
Fix error with newrelic.yml when running rake command for email updates

### ISSUE THREAD

**Issue by fma2** [BetaNYC/citygram-nyc#15]
Here's the error:
** [NewRelic][02/12/15 14:26:44 +0000 run.* (3)] ERROR : No application name configured.
** [NewRelic][02/12/15 14:26:44 +0000 run.* (3)] ERROR : The Agent cannot start without at least one. Please check your 
** [NewRelic][02/12/15 14:26:44 +0000 run.* (3)] ERROR : newrelic.yml and ensure that it is valid and has at least one 
** [NewRelic][02/12/15 14:26:44 +0000 run.* (3)] ERROR : value set for app_name in the production 
** [NewRelic][02/12/15 14:26:44 +0000 run.* (3)] ERROR : environment.

Basically, we need a value for app_name in the newrelic.yml file.

**Comment by dirkkelly** [comment]
Working on this now.
